roles/unbind-keycloak-apb/tasks/main.yml

```diff
@@ -35,4 +34,11 @@
       Authorization: "bearer {{ keycloak_auth_response.json.access_token }}"
     status_code: 204
     return_content: yes
-  when: keycloak_auth_response.status != 503
\ No newline at end of file
+  when: keycloak_auth_response.status != 503
+
+# Delete client secret
+- name: Delete keycloak-{{ _apb_bind_creds.client_name }} secret
+  k8s_v1_secret:
+    name: "{{ keycloak_secret_name }}-{{ CLIENT_CONFIG.resource | lower }}"
```

Comment:
I think `resource` should have more meaningful name. Its not obvious what it should hold